hs_core/hydroshare/users.py

```diff
@@ -595,7 +599,12 @@ def get_resource_list(creator=None,
 
             if user:
                 user = user_from_id(user)
-                queries[t].append(Q(edit_users=user) | Q(owners=user))
+                queries[t].append(Q(raccess__r2urp__privilege__lte=PrivilegeCodes.CHANGE,
+                                    raccess__r2urp__user=user.uaccess) |
+                                  Q(raccess__r2urp__privilege__lte=PrivilegeCodes.OWNER,
+                                    raccess__r2urp__user=user.uaccess)
+                                  )
+
```

Comment:
There is nothing in access control that does this. Closest match is u.get_owned_resources() 
But there should be. Consider writing u.get_changeable_resources() 

Also, this code is too literal and doesn't match the access control model. 
Correct query is (raccess__immutable=False, raccess__r2urp__privilege__lte=PrivilegeCodes.CHANGE)
The first phrase deals with immutability of a resource; the second deals with user privilege.